frontend/packages/kubevirt-plugin/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@console/kubevirt-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Kubevirt - Virtual machines addon for Kubernetes",
+  "private": true,
+  "dependencies": {
+    "@console/internal": "0.0.0-fixed",
+    "@console/plugin-sdk": "0.0.0-fixed",
+    "@console/shared": "0.0.0-fixed",
+    "kubevirt-web-ui-components": "^0.1.34-4"
+  },
+  "consolePlugin": {
+    "entry": "src/plugin.ts"
+  }
+}
+
```

Comment:
Nit-pick: extra trailing newline (one trailing newline should be the default for all sources, this should be checked by Prettier in future).